beacon-chain/core/blocks/withdrawals.go

```diff
@@ -148,6 +148,11 @@ func ValidateBLSToExecutionChange(st state.ReadOnlyBeaconState, signed *ethpb.Si
 //	        next_validator_index = ValidatorIndex(next_index % len(state.validators))
 //	        state.next_withdrawal_validator_index = next_validator_index
 func ProcessWithdrawals(st state.BeaconState, executionData interfaces.ExecutionData) (state.BeaconState, error) {
+	IsParentBlockFull, err := st.IsParentBlockFull()
+	if err != nil || !IsParentBlockFull {
+		return nil, errors.Wrap(err, "parent block is not full")
```

Comment:
This will fail on non-ePBS states. The logic should be to add 
```
    if st.Version() >= version.EPBS {
        do something
 ```
 or similar statements to keep the same previous functionality on pre-epbs states.